src/index.html

```diff
@@ -1,12 +1,366 @@
+        rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          class="header__logo-image"
+          src="./images/logo.png"
+          alt="Brand Logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active nav__link--active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <h1 class="visually-hidden">Catalog</h1>
+
+      <section class="cards">
+        <article class="card" data-qa="card">
+          <img
+            class="card__picture"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)"
+          >
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27&quot; Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-text">Reviews: 5</p>
+          </div>
+
+          <p class="card__price">
+            <span class="card__price-label">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </p>
+
+          <a
+            data-qa="card-hover"
+            class="card__button"
+            href="#"
+          >
+            Buy
+          </a>
+        </article>
```

Comment:
I am not sure, but I think section and article are not good choices for these cards especially for SEO, just div is better choice

Reply:
For semantics .